monkeytype/cli.py

```diff
@@ -147,9 +151,10 @@ def apply_stub_handler(args: argparse.Namespace, stdout: IO, stderr: IO) -> None
             src_path
         ])
         try:
-            subprocess.run(cmd, shell=True, check=True, stderr=subprocess.STDOUT, stdout=subprocess.PIPE)
+            proc_output = subprocess.run(cmd, shell=True, check=True, stderr=subprocess.STDOUT, stdout=subprocess.PIPE)
```

Comment:
Minor nit, but let's just call this variable `proc`, not `proc_output`; it is a `Process` object and has more than just "output" attributes on it (even though that's all we're using for  now).

Reply:
Sure

Reply:
Done.